usd/include/sdf/usd/Conversions.hh

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2022 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+
+#ifndef SDF_USD_CONVERSIONS_HH_
+#define SDF_USD_CONVERSIONS_HH_
+
+#include <memory>
+
+#include <ignition/common/Material.hh>
+
+#include "sdf/Material.hh"
+#include "sdf/sdf_config.h"
+#include "sdf/usd/Export.hh"
+
+namespace sdf
+{
+  // Inline bracket to help doxygen filtering.
+  inline namespace SDF_VERSION_NAMESPACE {
+  //
+  namespace usd
+  {
+    /// \brief Specialized conversion from an Ignition Common Material
+    /// to a SDF material
+    /// \param[in] _in Ignition Common Material.
+    /// \return SDF material.
+    IGNITION_SDFORMAT_USD_VISIBLE
+    sdf::Material convert(
+        const std::shared_ptr<ignition::common::Material> &_in);
```

Comment:
is a `const std::shared_ptr<Material>` what we want to pass in here? this will still allow mutation of the underlying data. Can we pass in a `const` raw pointer or `std::shared_ptr<const Material>` instead?

Reply:
I do think the `const`-ness of this pointer could still be adjusted

Reply:
I updated this API to use a `const ignition::common::Material *` in https://github.com/ignitionrobotics/sdformat/pull/831/commits/8b97a1c31c3714bd0f487d808058cb501ce4bae5, so a `shared_ptr` is no longer used.